<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connect_pushButton">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>21</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>disconnect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="show_state_browser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>881</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>20</y>
      <width>262</width>
      <height>175</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mqtt_message_verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Host_comboBox">
         <item>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>183.230.40.39</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>iot.eclipse.org</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Port_comboBox">
         <item>
          <property name="text">
           <string>1883</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6002</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IN_Usernae">
         <property name="text">
          <string>194413</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Client ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IN_Client_ID">
         <property name="text">
          <string>505578584</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IN_Password">
         <property name="text">
          <string>se3390</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Timeout:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IN_Timeout">
         <property name="text">
          <string>120</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Public_verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Toplic to Public:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Pushlish_pushButton">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Pushlish</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="Toplic_public_lineEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>QoS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Public_Qos">
         <item>
          <property name="text">
           <string>0 - Almost Once</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 - Atleast Once</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 - Exactly Once</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="Public_message_textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>201</y>
      <width>361</width>
      <height>289</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Subscribe_verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Toplic to subscribe:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Subscribe_pushButton">
         <property name="text">
          <string>Subscribe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="Subscribe_lineEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>QoS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Subscribe_Qos">
         <item>
          <property name="text">
           <string>0 - Almost Once</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 - Atleast Once</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 - Exactly Once</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="SubscribeMessage_Rece"/>
     </item>
    </layout>
   </widget>
   <zorder>connect_pushButton</zorder>
   <zorder>disconnect_pushButton</zorder>
   <zorder>Pushlish_pushButton</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>IN_Usernae</zorder>
   <zorder>label_4</zorder>
   <zorder>IN_Client_ID</zorder>
   <zorder>label_5</zorder>
   <zorder>IN_Password</zorder>
   <zorder>label_6</zorder>
   <zorder>IN_Timeout</zorder>
   <zorder>Host_comboBox</zorder>
   <zorder>Port_comboBox</zorder>
   <zorder>label_7</zorder>
   <zorder>Toplic_public_lineEdit</zorder>
   <zorder>label_8</zorder>
   <zorder>Public_Qos</zorder>
   <zorder>Public_message_textEdit</zorder>
   <zorder>label_9</zorder>
   <zorder>Subscribe_pushButton</zorder>
   <zorder>label_10</zorder>
   <zorder>Subscribe_Qos</zorder>
   <zorder>SubscribeMessage_Rece</zorder>
   <zorder>Subscribe_Qos</zorder>
   <zorder>Subscribe_lineEdit</zorder>
   <zorder>show_state_browser</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
